cluster/manifests/deletions.yaml

```diff
@@ -59,7 +59,7 @@ post_apply:
   kind: ClusterRoleBinding
 {{- end }}
 
-{{ if ne .Cluster.ConfigItems.skipper_ingress_canary_enabled "true" }}
+{{ if and (ne .Cluster.ConfigItems.skipper_ingress_canary_enabled "true") (ne .Cluster.ConfigItems.skipper_ingress_test_single_pod "true") }}
```

Comment:
We don't need this change. The controller to do load tests can also just set canary to false.

Reply:
I see your idea: loadtest controller must set both "skipper_ingress_canary_enabled" and "skipper_ingress_test_single_pod" to "true". Okay, it makes sense.